main/src/main/scala/sbt/internal/LibraryManagement.scala

```diff
@@ -19,6 +22,37 @@ private[sbt] object LibraryManagement {
 
   private type UpdateInputs = (Long, ModuleSettings, UpdateConfiguration)
 
+  def defaultUseCoursier: Boolean = {
+    val coursierOpt = sys.props
+      .get("sbt.coursier")
+      .flatMap(
+        str =>
+          ConsoleAppender.parseLogOption(str) match {
+            case LogOption.Always => Some(true)
+            case LogOption.Never  => Some(false)
+            case _                => None
+          }
+      )
+    val ivyOpt = sys.props
+      .get("sbt.ivy")
+      .flatMap(
+        str =>
+          ConsoleAppender.parseLogOption(str) match {
+            case LogOption.Always => Some(true)
+            case LogOption.Never  => Some(false)
+            case _                => None
+          }
+      )
+    val notIvyOpt = ivyOpt map { !_ }
+    coursierOpt.orElse(notIvyOpt).getOrElse(true)
```

Comment:
this default to true, are we confident enough to have it as the default choice?

Reply:
I think it's ready to at least try for 1.3.0-RC-1, and run it against Scala Community Build.